third_party/2and3/pytz/__init__.pyi

```diff
@@ -1,7 +1,7 @@
 # Stubs for pytz (Python 3.5)
 
 import datetime as dt
```

Comment:
Note that this line actually ends up exporting the name `dt` from the stub, while a plain `import datetime` would not (this is a quirk for stubs described in PEP 484).